frontends/mit-open/src/pages/SearchPage/SearchPage.tsx

```diff
@@ -113,9 +128,17 @@ export const getFacetManifest = (
       labelFunction: (key: string) => getDepartmentName(key) || key,
     },
   ]
+
+  //Only display the resource_type facet if the resource_category is learning_material
+  if (resourceCategory !== LEARNING_MATERIAL) {
+    mainfest.splice(1, 1)
+  }
+
+  return mainfest
 }
 
 const facetNames = [
+  "resource_type",
   "resource_category",
```

Comment:
I don't think `resource_category` should be a facetName, btw.
- if it is, then "Clear All" will reset the tab

I discussed this once with Steve and we decided "Clear all" should not clear the tab, since it's associated with the left-hand facets / filters.

If you change, resource_category would needed to be added o the aggregation separately.